File: PROGRAMMERS/lv0/두 수의 나눗셈.js
function solution(num1, num2) {
  return parseInt((num1 / num2) * 1000);
}

let inps = [
  [3, 2],
  [7, 3],
  [1, 16],
];

for (let i of inps) {
  console.log(solution(i[0], i[1]));
}

File: PROGRAMMERS/lv0/python/두 수의 나눗셈.py
def solution(num1, num2):
    return int(num1 / num2 * 1_000)

inps = [
    [3, 2],
    [7, 3],
    [1, 16]
]

for (n1, n2) in inps:
    print(solution(n1, n2))